src/actions/index.js

```diff
@@ -1,31 +1,37 @@
 // Movie key
-
 export const toggleFavorite = (id) => ({
-    type:'TOGGLE_FAVORITE',
-    id,
+  type: 'TOGGLE_FAVORITE',
+  id,
 })
 
 export const setMovies = (movies) => ({
   type: 'SET_MOVIES',
   movies,
 })
 
-export const setFavorites = (favorites, movies) => ({
-    type: 'SET_FAVORITES',
-    favorites,
-    movies,
+export const setFavorites = (favorites, user_id) => ({
+  type: 'SET_FAVORITES',
+  favorites,
+  user_id
 })
 
 // Users key 
 
 export const setUser = (user) => ({
-    type: 'SET_USER',
-    user,
+  type: 'SET_USER',
+  user,
 })
 
 // LoginPrompt key
 
-export const setPromptLogin = (bool) => ({
-  type: 'SET_PROMPT_LOGIN',
+export const setShouldPromptLogin = (bool) => ({
+  type: 'SET_SHOULD_PROMPT_LOGIN',
```

Comment:
When I read this the first time it seemed a little repetitive. Maybe we could just say 'SET_LOGIN_PROMPT' (flip login/prompt around from the original)? I'm wondering if that would convey that it's either true or false in the same way as well as including the word should.

Reply:
#40